kratos/utilities/math_utils.h

```diff
@@ -429,12 +432,16 @@ class MathUtils
         const TDataType Tolerance = ZeroTolerance
         )
     {
-        const SizeType size = rInputMatrix.size2();
+        const SizeType size = rInputMatrix.size1();
```

Comment:
why is this changed from size2 to size1?

Reply:
I do not remember, too much time ago, but I know it was for a reason. Probably in order to consider not square matrices